src/sentry/configurations/docs/api.md

```diff
@@ -0,0 +1,72 @@
+# Configurations API
+
+Host: https://sentry.io/api/0
+
+**Authors.**
+
+@cmanallen
+
+## Configuration [/projects/<organization_id_or_slug>/<project_id_or_slug>/configuration/]
+
+### Get Configuration [GET]
+
+Retrieve the project's configuration.
+
+**Attributes**
+
+| Column             | Type   | Description                                           |
+| ------------------ | ------ | ----------------------------------------------------- |
+| id                 | number | The id of the project's configuration being returned. |
+| sample_rate        | number | A number value between 0 and 1.                       |
+| traces_sample_rate | number | A number value between 0 and 1.                       |
+| user_config        | any    | Arbitrary user supplied JSON.                         |
+
+- Response 200
+
+  ```json
+  {
+    "data": {
+      "id": 1,
```

Comment:
`id`?

Reply:
The ID of the project whose configuration you're reading.  Should have been more clear in the table above.  We can remove it.

Reply:
Eh. Looks like it's just passed along. I'm unbothered.